src/components/shared/LessonSide/LevelSelector.tsx

```diff
@@ -1,5 +1,31 @@
-function LevelSelector(): JSX.Element {
-  return <div>Level Selector</div>;
+import '../../../styles/LessonSide.scss';
+
+interface IProps {
```

Comment:
We should try to follow `(componentType)Props` so that if we need to export the interface, it's easy to know which props the interface relates to
```suggestion
interface LevelSelectorProps {
```